wagtail/users/templates/wagtailusers/users/results.html

```diff
@@ -15,7 +15,7 @@ <h2 role="alert">
     {% include "wagtailusers/users/list.html" %}
 
     {# call pagination_nav with no linkurl, to generate general-purpose links like <a href="?p=2"> #}
-    {% include "wagtailadmin/shared/pagination_nav.html" with items=users %}
+    {% include "wagtailadmin/shared/pagination_nav.html" with items=paginator %}
```

Comment:
As above, this should be `page_obj`

```suggestion
    {% include "wagtailadmin/shared/pagination_nav.html" with items=page_obj %}
```


This gets used in https://github.com/wagtail/wagtail/blob/main/wagtail/admin/templates/wagtailadmin/shared/pagination_nav.html

Here, also, the name is a bit confusing but you can see in the template it is reading out `has_previous` and `number` which are on the `paginator.Page` instance.

https://docs.djangoproject.com/en/4.1/ref/paginator/#django.core.paginator.Page

Reply:
Got it! I am going to make all the requested changes. @lb- @gasman thanks a lot for your feedback and time.